agent/test/README.md

```diff
@@ -0,0 +1,46 @@
+# Test for HeapStats
+
+Test cases for HeapStats. These test cases could run on containers.
+
+## Usage
+
+### on container
+
+```
+docker build --pull -t heapstats/$(basename $PWD) -f ./Dockerfile(.el7) .
```

Comment:
Please change tag name.
If you want to use directory basename as the tag, please add the reason.